notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb

```diff
@@ -0,0 +1,1229 @@
+{
```

Comment:
<p>Use the method:</p><p>delete_bucket = False</p><p> </p><p>if delete_bucket or os.getenv("IS_TESTING"):</p><p>    ! gsutil -m rm -r n$BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>